screens/DayScreen.tsx

```diff
@@ -39,13 +137,10 @@ const DayScreen: React.FC<DayInfo> = (dayInfo: DayInfo) => {
     }
   }, [data]);
 
-  const onRefresh = async () => {
-    await refreshContext();
-  };
-
   return (
-    <View style={styles.container}>
+    <>
```

Comment:
Er zit hier ergens nog eenzelfde bug als bij de andere schermen, er is een margin waardoor de timeline vroeger afsnijdt

Reply:
still the same for me